### PR TITLE
feat : added Navbar

### PR DESCRIPTION
**Pull request by Priya-Patel31**

### CODE REVIEW

**Review by Omkar-Chaskar — APPROVED** [review]
Every thing is good ,
Nice UI , Good to merge

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  --primary-color: #fc3936;
+  --background-color: #121212;
+  --secondary-color: #070708;
+  --text-color: #f1efef;
+  --accent-color: #3700b3;
+  --success-color: #3ded1d;
+  --error-color: #f20b07;
+  --shadow-color: #0066ff;
+  --search-bar-color: rgb(221, 221, 221);
+  --dark-color: black;
```
Comment (Ankur9669) [inline src/App.css:14]: Suggestion: Can use hex code here

**Review by Ankur9669 — APPROVED** [review]
The Code looks good to merge

File: src/App.css (hunk 1 above)
Comment (Priya-Patel31) [reply src/App.css:14]: I will change.

**Pull request merged**